app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def comments_count
+    post.increment!(:comments_count)
+  end
```

Comment:
- [ ] Use the existing attribute `Comments_counter` instead of `comments_count`. The above implementation will output the `unknown attribute 'comments_count' for Post`